src/lib/codeSamples/request.ts

```
@@ -0,0 +1,9 @@
+export class OARequest {
+  constructor(
+    public url: string = '',
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)